src/activity_item_handling.cpp

```diff
@@ -513,63 +504,88 @@ static std::list<act_item> reorder_for_dropping( Character &p, const drop_locati
                     && !second.loc->is_worn_only_with( *first.loc ) );
     } );
 
-    // Cumulatively increases
-    units::volume storage_loss = 0_ml;
+    // Avoid tumbling to the ground. Unload cleanly.
+    units::volume dropped_inv_contents = std::accumulate( inv.begin(), inv.end(), 0_ml,
+    []( units::volume acc, const act_item & ait ) {
+        return acc + ait.loc->volume();
+    } );
+    const units::volume dropped_worn_storage = std::accumulate( worn.begin(), worn.end(), 0_ml,
+    []( units::volume acc, const act_item & ait ) {
+        return acc + ait.loc->get_storage();
+    } );
+    std::set<int> inv_indices;
+    std::transform( inv.begin(), inv.end(), std::inserter( inv_indices, inv_indices.begin() ),
+    [&p]( const act_item & ait ) {
+        return p.get_item_position( &*ait.loc );
+    } );
+
+    units::volume excessive_volume = p.volume_carried() - dropped_inv_contents
+                                     - p.volume_capacity() + dropped_worn_storage;
```

Comment:
```suggestion
    units::volume excessive_volume = p.volume_carried() - dropped_inv_contents
                                     - p.volume_capacity_reduced_by( dropped_worn_storage );
```
`volume_capacity_reduced_by` is aware of bionics/traits/mutations, so this should fix the Packmule bug